usaspending_api/etl/management/commands/update_file_c_linkages.py

```diff
@@ -9,28 +9,75 @@
 
 class Command(BaseCommand):
 
+    help = (
+        "By default, this command will use the `c_to_d_linkage_updates` table to determine which FABA",
+        "records need to be updated. This table is populated by the `update_file_c_linkages_in_delta`",
+        "command which should be run before this command during the nightly pipeline. It contains a",
+        "mapping from FABA records to Awards. However, if the --recalculate-linkages flag is used, the",
+        "necessary updates will be reculated using a series of SQL files",
+    )
+
+    UPDATE_LINKAGES_SQL = """
+    UPDATE
+        public.financial_accounts_by_awards AS faba
+    SET
+        award_id = updates.award_id
+    FROM
+        public.c_to_d_linkage_updates AS updates
+    WHERE
+        faba.financial_accounts_by_awards_id = updates.financial_accounts_by_awards_id;
+    """
```

Comment:
I think it would be good here to only update the FABA records that need updating, so add an `AND faba.award_id IS DISTINCT FROM updates.award_id` would be helpful for performance. 

I still need to review further though if this sis necessary, or if the updates table only holds the changes in linkages from the last run.

Reply:
I was thinking about something like this, but the table does indeed only hold updates. Shouldn't be any need to add this extra check